### DISCUSSION TITLE
Feature request: handle dashboard "requests" before other tasks

### DISCUSSION THREAD

**Discussion by dylang** [renovatebot/renovate#30022] [Suggest an Idea]
### Situtation

Sometimes we need a dependency PR from `Rate-Limited` and don't care about other Renovate PRs being updated first.

We're using self-hosted, and no web hooks.

### Background

When Renovate runs, it seems to look the checked boxes on the dashboard **after** doing all other tasks first. So seems to be updating open PRs and opening new PRs first, which takes about 30 minutes for our larger repos. (Sometimes it doesn't even do any of the checked items on dashboard, like it ran out of time. There's nothing in the `info` log that explains why.)

It seems like this is the order:

1. Merge PRs.
2. Update out-of-date PRs.
3. Open new PRs.
4. Look at dashboard for PRs to open or rebase.
5. Update Dashboard.

### Feature Request

We'd like if the dashboard was handled first, for those times we need a PR opened quickly.

**Comment by rarkins** [comment]
I'm pretty sure that Renovate just goes in the regular order of sorted branches, which is typically in order of digest, patch, minor, major. It just so happens that those which are first in the order are the ones likely to have existing PRs, so it looks like Renovate prioritizes updating out-of-date PRs first.

**Comment by rarkins** [comment]
Converted to feature request: https://github.com/renovatebot/renovate/issues/30026
